### DISCUSSION TITLE
Unable to deploy discord bot on vercel

### DISCUSSION THREAD

**Discussion by theabhayprajapati** [vercel/vercel#7767] [Help] [answered]
I was trying to deploy a discord bot on vercel but it was getting me some serve error

<img width="960" alt="image" src="https://user-images.githubusercontent.com/88815641/167339064-d247db68-c115-435a-8f84-fed89cbcf668.png">

Help me to solve problem

**Comment by leerob** [comment, marked as answer]
Hey! Based on your build output, I see you're using a Node.js server. Vercel is a **serverless platform**, which means we don't support the deployment of long-running Node.js servers, commonly used for things like Discord bots.

If you don't want to change anything with your code, consider deploying to a service like [Railway](https://railway.app/). Alternatively, you could probably write your Discord bot using [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) to make it work on Vercel.

**Reply by theabhayprajapati** [reply]
Hey! @leerob thank you actually I am following you for the last few months getting a response from you is really amazing; I didn't know about [Railway ](https://railway.app) thanks for sharing it solved my problem.

Thank you
